cg/clients/freshdesk/models.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class Ticket(BaseModel):
+    """Freshdesk ticket."""
+
+    association_type: int | None = None
+    attachments: list[dict[str, str]] = []
+    cc_emails: list[str] = []
+    company_id: int | None = None
+    custom_fields: dict | None = None
+    deleted: bool | None = None
+    description: str | None = None
+    description_text: str | None = None
+    due_by: datetime | None = None
+    email: str | None = None
+    email_config_id: int | None = None
+    facebook_id: str | None = None
```

Comment:
are we ever going to use this?

Reply:
Nope, I just added everything the return-model contains

Reply:
removed in [3dca545](https://github.com/Clinical-Genomics/cg/pull/3317/commits/3dca545ffb56567f9685a269e81d154581de9f88)